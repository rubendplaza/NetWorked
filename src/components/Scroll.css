.cardListContainer {
    overflow-y: scroll;
    border-top: 2px solid #1F2833;
    position: absolute;
    top: 200px;
    bottom: 0;
    width: 100%;
}