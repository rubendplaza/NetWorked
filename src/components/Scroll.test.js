import React from "react";
import Scroll from "./Scroll";
import { shallow } from "enzyme";

test("Scroll component renders correctly without crashing", () => {
  expect(shallow(<Scroll />)).toMatchSnapshot();
});
