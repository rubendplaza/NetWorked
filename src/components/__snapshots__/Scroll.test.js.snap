// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll component renders correctly without crashing 1`] = `
<div
  className="cardListContainer"
/>
`;
